tests/unit/test__pandas_helpers.py

```diff
@@ -734,6 +743,37 @@ def test_list_columns_and_indexes_with_named_index_same_as_column_name(
     assert columns_and_indexes == expected
 
 
+@pytest.mark.skipIf(
```

Comment:
OK now I see it, I hate this gotcha. :)

```suggestion
@pytest.mark.skipif(
```